examples/todo/src/tests.rs

```diff
@@ -40,7 +40,7 @@ fn test_insertion_deletion() {
             .dispatch().await;
 
         // Ensure we have one more task in the database.
-        let new_tasks = Task::all(&conn);
+        let new_tasks = conn.run(|c| Task::all(c)).await;
```

Comment:
Does `conn.run(Task::all)` work?

Reply:
No for two reasons: `run()` gives an `&mut PooledConnection<...>` (fixing now), but even after that `Task::all` wants an `&` but `run` provides an `&mut` and there's no automatic coercion here. Some databases *need* the `&mut` option, so we would have to provide two versions of `run` to get this convenience back for functions that want a shared reference instead of a mutable one.